### ISSUE TITLE
Limit number of videos to import per channel

### ISSUE THREAD

**Issue by dayologun** [Elijas/auto-youtube-subscription-playlist-2#76]
Is there a way to limit the number of videos to import per channel? I have a playlist which videos are added from different channels. some of the channels have too many videos so i would like to import just a certain number from them so that my quota doesnt get used up by only one channel. I will appreciate your help!

**Comment by victorjzsun** [comment]
That is doable, although it will require an overhaul to the spreadsheet once again (maybe we need to find a better way to specify special arguments). I'd assume you'd want the latest n videos from the channel if we do filter it.

**Comment by dayologun** [comment]
Yes. The latest videos only. The ability to select a specific number of the latest videos in the channel.
